doc/source/roadmap.rst

```diff
@@ -98,11 +98,13 @@ however; we need to make it faster and make it easier to compare performance of
 optimizers via plotting performance profiles.
 
 
-Linear programming enhancements
--------------------------------
-
-Recently all known issues with ``optimize.linprog`` have been solved.  Now we
-have many ideas for additional functionality (e.g. integer constraints, sparse
-matrix support, performance improvements), see gh-9269.
+Statistics enhancements
+-----------------------
 
+The `scipy.stats` enhancements listed in the :ref:`scipy-roadmap-detailed` are of
+particularly high importance to the project.
 
+- Improve the options for fitting a probability distribution to data.
+- Expand the set of hypothesis tests.  In particular, include all the basic
```

Comment:
There are lots of non-parametric versions of the tests that could be included as well, the Hollander and Wolf book is basically an encyclopedia or listing of many of these methods, how to perform and when to use. Not sure if you want to include in scope or not.

Reply:
That sounds like something that could be added to the roadmap in another pull request after a bit more research.